packages/nc-gui/components.d.ts

```diff
@@ -90,6 +90,7 @@ declare module '@vue/runtime-core' {
     LogosMysqlIcon: typeof import('~icons/logos/mysql-icon')['default']
     LogosPostgresql: typeof import('~icons/logos/postgresql')['default']
     LogosRedditIcon: typeof import('~icons/logos/reddit-icon')['default']
+    LogosSnowflakeIcon: typeof import('~icons/logos/snowflake-icon')['default']
```

Comment:
I think ESLint added this line, which should be alright?

Reply:
Yes. This was introduced from another PR. Maybe the author didn't commit this change.